processor/apmprocessor/go.mod

```diff
@@ -18,6 +19,7 @@ require (
 	github.com/json-iterator/go v1.1.12 // indirect
 	github.com/knadh/koanf v1.5.0 // indirect
 	github.com/knadh/koanf/v2 v2.0.1 // indirect
+	github.com/lightstep/go-expohisto v1.0.0 // indirect
```

Comment:
kind of weird that the apm processor needs to know about the histogram library 🤔 it's probably because it now depends on the apm connector.  We should probably review this dependency